core/arch/arm/plat-versal2/platform_config.h

```diff
@@ -0,0 +1,59 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2023-2024, Advanced Micro Devices, Inc. All rights reserved.
+ */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+#include <mm/generic_ram_layout.h>
+
+/* Make stacks aligned to data cache line length */
+#define CACHELINE_LEN		64
+#define STACK_ALIGNMENT		CACHELINE_LEN
+
+#if defined(PLATFORM_FLAVOR_generic)
+
+#define PLM_RTCA		0xF2014000
+#define PLM_RTCA_LEN		0x1000
+
+#define GICD_BASE		U(0xE2000000)
+#define GICR_BASE		U(0xE2060000)
+
+#define UART0_BASE             U(0xF1920000)
+#define UART1_BASE             U(0xF1930000)
+
+#define IT_UART0		50
+#define IT_UART1		51
+
+#define UART0_CLK_IN_HZ		100000000
+#define UART1_CLK_IN_HZ		100000000
+#define CONSOLE_UART_BASE	UART0_BASE
+#define IT_CONSOLE_UART		IT_UART0
+#define CONSOLE_UART_CLK_IN_HZ	UART0_CLK_IN_HZ
+
+#define DRAM0_BASE		0
+#define DRAM0_SIZE		0x80000000
+
+#ifndef ARM64
+#error "Only ARM64 is supported!"
+#endif
+
+#else
+#error "Unknown platform flavor"
+#endif
+
+#ifdef CFG_TEE_LOAD_ADDR
+#define TEE_LOAD_ADDR			CFG_TEE_LOAD_ADDR
+#else
+#define TEE_LOAD_ADDR			TEE_RAM_START
+#endif
```

Comment:
these are already handled in [*generic_ram_layout.h*](https://github.com/OP-TEE/optee_os/blob/4.4.0/core/arch/arm/include/mm/generic_ram_layout.h#L114) and [*core_mmu.h*](https://github.com/OP-TEE/optee_os/blob/4.0.0/core/include/mm/core_mmu.h#L49)